src/GraphQL.Conventions/Types/Descriptors/GraphFieldInfo.cs

```diff
@@ -12,6 +13,7 @@ public class GraphFieldInfo : GraphEntityInfo
         public GraphFieldInfo(ITypeResolver typeResolver, MemberInfo field = null)
             : base(typeResolver, field)
         {
+            IsExtensionMethod = (AttributeProvider as MethodInfo)?.IsExtensionMethod() ?? false;
```

Comment:
Can bring the null-check down to the extension method:

```cs
public static bool IsExtensionMethod(this MethodInfo methodInfo) => 
    methodInfo?.IsDefined(typeof(ExtensionAttribute), false) ?? false;
```

And thus, this line could be removed and the property getter expressed like so:

```cs
public bool IsExtensionMethod => (AttributeProvider as MethodInfo).IsExtensionMethod()
```

#nitpick